docs/API.md

```diff
@@ -675,6 +676,45 @@ exports[`drinking flavors throws on octopus 1`] = `"yuck, octopus flavor"`;
 
 Check out [React Tree Snapshot Testing](http://facebook.github.io/jest/blog/2016/07/27/jest-14.html) for more information on snapshot testing.
 
+
+### Extending Jest Matchers
+
+Using descriptive matchers will help your tests be readable and maintainable. Jest has a simple API for adding your own matchers. Here is an example of adding a matcher:
+
+```js
+const five = require('five');
+
+expect.extend({
+  toBeNumber(received, actual) {
+    const pass = received === actual;
+    const message = `expected ${received} ${pass ? 'not ' : ''} to be ${actual}`;
+    return {message, pass};
+  }
+});
```

Comment:
Can you please document that it might be better to extend expect globally when it's supposed to be used across several test suites? You can do that in `setupTestFrameworkScriptFile` (quite sure it's possible) and possibly even `setupFiles` (not sure about this one).

Docs for the options:
http://facebook.github.io/jest/docs/configuration.html#setuptestframeworkscriptfile-string